Services/Administration/OJS.Services.Administration.Data/Implementations/SubmissionsDataService.cs

```diff
@@ -38,9 +38,11 @@ public IQueryable<Submission> GetAllByProblem(int problemId)
         public IQueryable<Submission> GetAllByProblems(IEnumerable<int> problemIds)
             => this.GetQuery(s => problemIds.Contains(s.ProblemId));
 
-        public Task<Submission?> GetWithProblemTestsAndSubmissionTypes(int id)
+        public Task<Submission?> GetNonDeletedWithNonDeletedProblemTestsAndSubmissionTypes(int id)
         {
-            var queryable = this.GetByIdQuery(id);
+            var queryable = this.GetByIdQuery(id)
+                .Include(s => s.Problem)
```

Comment:
This Include is not needed here, as it's already included in the next method

Reply:
Fixed